deploy/crds/shipwright.io_buildruns.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.4.1
+    controller-gen.kubebuilder.io/version: v0.5.0
```

Comment:
can u bump the version in the installation script https://github.com/shipwright-io/build/blob/main/hack/install-controller-gen.sh#L14 pls

Reply:
Done.